### PR TITLE
Fix #1558 status bar bug fixed, status bar doesnt reappear on opening spinners

### LINKED ISSUE #1558: StatusBar appears in Oscilloscope and Logic Analyzer on opening any drop down list
**Actual Behaviour**

Upon expanding any drop down list in oscilloscope or logic analyzer , status bar reappears.

**Expected Behaviour**

status bar should be hidden

**Steps to reproduce it**

Open Oscilloscope/ Logic Analyzer, expand any drop down list

**Screenshots of the issue**

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by neel1998**
Fixes #1558  
**Changes**: widow full screen flag set in onCreate method

**Screenshot/s for the changes**: 
screenshot_20190220-165309

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [ ] I have requested reviews from other members

**APK for testing**: 
[statusbar_bug.zip](https://github.com/fossasia/pslab-android/files/2884414/statusbar_bug.zip)

### CODE REVIEW

File: app/src/main/java/io/pslab/activity/OscilloscopeActivity.java [hunk 1]
```diff
@@ -396,6 +400,8 @@ public void run() {
         monitorThread.start();
     }
 
+
```
Comment (CloudyPadmal) [inline app/src/main/java/io/pslab/activity/OscilloscopeActivity.java]: Can you please remove these blank lines? Rest looks good

**Comment by neel1998** [comment]
@CloudyPadmal  kindly review. I have made the changes.

**Comment by neel1998** [comment]
@CloudyPadmal  i resolved the conflict ..kindly review

**Comment by neel1998** [comment]
@CloudyPadmal  kindly review and merge.

**Pull request merged**